server/schema/Resolvers.ts

```diff
@@ -198,9 +214,8 @@ const resolvers: IResolvers = {
         ): Promise<ProjectQuery | Error> => {
```

Comment:
What's the business case for creating a query via the app? The way I've been thinking about it the app is just a visualization tool for queries that are already in the database and queries would be added to the database by the logging tool rather than through the UI? That being said this logic would definitely need to exist in the logging package

Reply:
This is the logic for the logger to add the queries to the app. The logger will be sending mutation requests to our apollo backend through axios/fetch

Reply:
Can the logger and the app just use a shared database for queries? If I'm understanding what you're saying correctly the flow would look something like: 

- logger middleware intercepts query
- logger sends request to app server
- app server logs query in database
- app reads query from database and uses it in visualization

I think we can cut out a step and just do something like
- logger intercepts query
- logger records query in database
- app reads query info from database